### PR TITLE
feat(mlh): let users configure the official MLH theme

### PR DESCRIPTION
**Pull request by bartekpacia**
## Standards checklist:

- [x] The PR title is descriptive.
- [x] The PR doesn't replicate another PR which is already open.
- [x] I have read the contribution guide and followed all the instructions.
- [x] The code follows the code style guide detailed in the wiki.
- [x] The code is mine or it's from somewhere with an MIT-compatible license.
- [x] The code is efficient, to the best of my ability, and does not waste computer resources.
- [x] The code is stable and I have tested it myself, to the best of my abilities.

## Changes:

- let users configure MLH theme using environmental variables. User can set
  them conveniently, for example, in their `~/.zshrc` file, instead of being
  forced to override the whole theme.

- make the theme 2 line. Currently, when the user is working on a branch with a long name, it looks
  really awkward:

Screenshot 2021-06-21 at 13 09 08

Now, it looks just cool.
Screenshot 2021-06-21 at 22 09 10

### CODE REVIEW

File: themes/mlh.zsh-theme [hunk 1]
```diff
@@ -12,17 +12,38 @@
 # # # Feel free to customize! # # #
 # # # # # # # # # # # # # # # # # #
 
-# To easily discover colors and their codes, type `spectrum_ls` in the terminal
+# To customize symbols (e.g MLH_AT_SYMBOL), simply set them as environment variables
+# for example in your ~/.zshrc file, like this:
+# export MLH_AT_SYMBOL=" at "
```
Comment (mcornella) [inline themes/mlh.zsh-theme:17]: No need to use `export`. Also you should document that the settings need to go before OMZ is sourced in the `.zshrc` file.

File: themes/mlh.zsh-theme [hunk 2]
```diff
@@ -47,24 +68,28 @@ directory() {
 
 # Prints current time
 current_time() {
-  if [ "$PRINT_TIME" = true ]; then
-    echo " $TIME_COLOR%*%f"
+  if [ "$MLH_PRINT_TIME" = true ]; then
+    echo " $MLH_PRINT_TIME%*%f"
```
Comment (mcornella) [inline themes/mlh.zsh-theme:72]: Typo?

**Review by mcornella — CHANGES_REQUESTED** [review]

File: themes/mlh.zsh-theme (hunk 2 above)
Comment (bartekpacia) [reply themes/mlh.zsh-theme:72]: yes, thank you:)

File: themes/mlh.zsh-theme (hunk 1 above)
Comment (bartekpacia) [reply themes/mlh.zsh-theme]: thanks:)

**Review by mcornella — APPROVED** [review]

**Comment by mcornella** [comment]
Thanks Bartek!

**Pull request merged**